src/CallHandler.js

```diff
@@ -179,7 +216,6 @@ function _setCallState(call, roomId, status) {
 function _onAction(payload) {
     function placeCall(newCall) {
         _setCallListeners(newCall);
-        _setCallState(newCall, newCall.roomId, "ringback");
```

Comment:
and this?

Reply:
still wondering why this is going away

Reply:
Ah, because playing the ringback tone ought to indicate that the call is at least probably ringing at the other end, so we set it when the invite has been sent, rather than starting it as soon as we place the call.

Reply:
where does that happen?